doc/GithubPRs.md

```diff
@@ -0,0 +1,37 @@
+GitHub PRs workflow
+======
+
+See also https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
+
+Draft PRs
+------
+
+A PR should be marked as `draft` if it is not intended to be merged as is, but is waiting for some sort of feedback.
+The author of the PR should explicit what kind of feedback is expected (CI/QA run, discussion on the code, etc...)
+
+GitHub filter is `is:pr is:open draft:true sort:updated-asc`
+
+A draft may be closed if it has not been updated in the last month.
```

Comment:
Idea: "A draft may be automatically closed"

We could easily script this, and I think we should at some point.

Reply:
indeed